### ISSUE TITLE
Pull requests: How to

### ISSUE THREAD

**Issue by nicrusso7** [SmartThingsCommunity/SmartThingsPublic#2475]
Hi guys!
Would be great if I could update my SmartApp (Gideon Smart Home) and add new features.
How can I do it? I've tried with the pull request but I didn't get any approval..

Thanks!

**Comment by bbodenmiller** [comment]
Since they seem to be totally non-responsive on GitHub perhaps send an email at https://support.smartthings.com/hc/en-us?

**Comment by workingmonk** [comment]
@NicolaRusso can you put the link to the PR here?

**Comment by workingmonk** [comment]
cc @tslagle13

**Comment by nicrusso7** [comment]
@workingmonk here the link: https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2452/commits/880c9e9f7334004e368aeb7110881de0ee6b4940

It's just a request to delete old code. I also have new code to deploy but I don't understand how to.

Thanks,

Nico

**Comment by workingmonk** [comment]
you want to delete your integration?

**Comment by nicrusso7** [comment]
@workingmonk no, I made a mistake during the first release. Basically here the repo master: https://github.com/SmartThingsCommunity/SmartThingsPublic/tree/master/smartapps/gideon-api

It has 2 folder, this one is the current (and correct) smartapp: https://github.com/SmartThingsCommunity/SmartThingsPublic/tree/master/smartapps/gideon-api/gideon-smart-home.src

The other one (https://github.com/SmartThingsCommunity/SmartThingsPublic/tree/master/smartapps/gideon-api/gideon.src) is an old one that I want to delete.

Anyway I also have a new version that includes new devices and would be great merge it as well.

**Comment by workingmonk** [comment]
@tslagle13 can you help with this? we need to make sure no one is using the smartapp that needs to be deleted. after we delete the file from the repo, we'll also have to go ahead and delete it from superuser in all the shards

**Comment by nicrusso7** [comment]
@workingmonk @tslagle13 thanks for the support. I made another pull request to add new devices here https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2492/commits/b8be6213dbbdf1196d9d1d4d033f8a0aa91fbe34

**Comment by nicrusso7** [comment]
@workingmonk @tslagle13 Guys, If I can help in some way just tell me what I have to do! or maybe you can give me a name/contact from business development (I really don't know how to find them! already tried on Linkedin) so I can try to get an approval or something to speed up everything..
